client/extensions/zoninator/state/data-layer/zones/index.js

```diff
@@ -77,18 +104,61 @@ export const announceZoneCreated = ( { dispatch, getState }, action, response )
 	announceZoneSaved( dispatch, action, response.data );
 };
 
-export const announceFailure = ( { dispatch }, { form } ) => {
+export const announceZoneUpdated = ( { dispatch, getState }, action ) => {
```

Comment:
I'm still not sure I totally agree with calling this `announceZoneUpdated` since the notice isn't being dispatched directly from inside this function. It's really resetting the dirty state of the form and then calling another function that handles the actual announcement.

So what to call this one then? `initializeForm`? `resetForm`? `resetAndAnnounceZoneUpdated`? I'm not too sure what's best. Naming is hard! ☹️